<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlotSetupWidget</class>
 <widget class="QWidget" name="PlotSetupWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>356</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="1">
    <widget class="QPushButton" name="swapXYAxis">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="accessibleName">
      <string>Swap</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="toolTip">
      <string>Swap X and Y axis</string>
     </property>
     <property name="icon">
      <iconset theme="swap-panels"/>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>X-Axis</string>
     </property>
     <layout class="QGridLayout">
      <property name="margin">
       <number>9</number>
      </property>
      <property name="spacing">
       <number>6</number>
      </property>
      <item row="3" column="0">
       <widget class="QLabel" name="textLabel1_2">
        <property name="text">
         <string>Average value:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="minimum_x">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="av_x">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="KComboBox" name="KCB_x">
        <property name="whatsThis">
         <string>Here you can define what you want to plot</string>
        </property>
        <item>
         <property name="text">
          <string>Atomic Number</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Atomic Mass</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Electronegativity</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Melting Point</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Boiling Point</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Atomic Radius</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Covalent Radius</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="textLabel2">
        <property name="text">
         <string>Maximum value:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="maximum_x">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="textLabel3_2">
        <property name="text">
         <string>Minimum value:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="3">
    <layout class="QGridLayout" name="gridLayout3">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item row="3" column="0">
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Display:</string>
       </property>
       <property name="buddy">
        <cstring>comboElementLabels</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Element Type:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="to">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimum">
        <number>2</number>
       </property>
       <property name="maximum">
        <number>111</number>
       </property>
       <property name="value">
        <number>36</number>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="textLabel3">
       <property name="text">
        <string>First element:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="textLabel4">
       <property name="text">
        <string>Last element:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="from">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>110</number>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="full">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>All elements</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="comboElementType">
       <item>
        <property name="text">
         <string>All elements</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Metals</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Non-Metals / Metalloids</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>s block elements</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>p block elements</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>d block elements</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>f block elements</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Noble gases</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Alkalie metals</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Alkaline earth metals</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Lanthanides</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Actinides</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Radio-active elements</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="KComboBox" name="comboElementLabels">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>No Labels</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Element Names</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Element Symbols</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Y-Axis</string>
     </property>
     <layout class="QGridLayout">
      <property name="margin">
       <number>9</number>
      </property>
      <property name="spacing">
       <number>6</number>
      </property>
      <item row="3" column="1">
       <widget class="QLabel" name="av_y">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="textLabel1_3">
        <property name="text">
         <string>Average value:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="maximum_y">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="textLabel2_2">
        <property name="text">
         <string>Maximum value:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="textLabel3_3">
        <property name="text">
         <string>Minimum value:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="minimum_y">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="KComboBox" name="KCB_y">
        <property name="whatsThis">
         <string>Here you can define what you want to plot</string>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <item>
         <property name="text">
          <string>Atomic Number</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Atomic Mass</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Electronegativity</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Melting Point</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Boiling Point</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Atomic Radius</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Covalent Radius</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="KPlotWidget" name="plotwidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>150</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KPlotWidget</class>
   <extends>QFrame</extends>
   <header>kplotwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>KCB_x</tabstop>
  <tabstop>KCB_y</tabstop>
  <tabstop>from</tabstop>
  <tabstop>to</tabstop>
  <tabstop>full</tabstop>
  <tabstop>comboElementLabels</tabstop>
  <tabstop>comboElementType</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
